fhir-install/src/main/docker/ibm-fhir-schematool/README.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: default
+title:  IBM FHIR Server - Schema Tool
+date:   2020-11-11
+permalink: /ibm-fhir-server-schema-tool/
+---
+
+# IBM FHIR Server - Schema Tool
+
+The IBM FHIR Server Schema Tool is designed to create and update the IBM FHIR Server's schema idempotently.
+
+The tool supports the following flows in one image: 
+
+* Onboarding - create SQL schemas, populate the SQL objects, grant permissions to the database user. For db2, the tool sets up a single tenant and tenant key. 
+* Offboarding - removes the schema, on Db2, if no other tenants, it deallocates the tenants
+* Custom - executes a single fhir-persistence-schema cli action
+* Debug - outputs debug information
+
+# **Use**
+
+The solution should set the variables as Environment Variables or mounted into the workarea folder as a volume which contains the persistence.json file.
+
+## **Environment Variables**
+The following environment variables:
+
+| Name           | Purpose  |
+|----------------|----------|
+| ENV_SKIP       | Stop the container from making any changes, and passes through with a successful state change: `[empty|true|false]`|
+| ENV_TOOL_INPUT | Encoded String, base64, of the json|
+| ENV_TOOL_DEBUG | Flags the debug |
+
+## Configuration Commandline
+The following is read from the properties file:
+
+| Name       | Purpose  |
+|------------|----------|
+| tool.behavior | Switches from the Onboarding BEHAVIOR to Offboarding BEHAVIOR flow: `[onboard|offboard|debug|custom]`. |
+| db.type | The database type - `[postgres|db2]` |
+| db.host | The database server hostname|
+| db.port | The database server port|
+| db.database | The name of the database|
+| user | A username with connect and admin permissions on the target database|
+| password | The user password for connecting to the database|
+| sslConnection | true or anything else, true triggers JDBC to use ssl, an example --prop sslConnection=true (db2) |
+| ssl | true or anything else, true triggers JDBC to use ssl, an example --prop ssl=true (postgres) |
+| tenant.key | the tenant key is a generated key, or used to set a specific value |
+| tenant.name | the tenant name is typically default |
+| schema.name.oauth | uses the default or custom |
+| schema.name.fhir | defaults to fhirdata |
+| schema.name.batch | uses the default or custom |
+| grant.to | grants access to a specific user (which is going to run the application) |
+
+Further, any property supported by the [fhir-persistence-schema](https://github.com/IBM/FHIR/blob/master/fhir-persistence-schema/README.md) module is put into the file and mounted to the system.
```

Comment:
is it worth explicitly listing the postgresql properties?
```
  --prop "sslmode=verify-full"
  --prop "sslrootcert=./postgres.cert"
```

Reply:
you bet, I'll add it.

Reply:
fixed, I'll push some fixes shortly.